docs/guides/python/ibis.md

```diff
@@ -3,147 +3,706 @@ layout: docu
+├─────────┼───────────┼───────┤
+│ Adelie  │ Torgersen │  2007 │
+│ Adelie  │ Torgersen │  2007 │
+│ Adelie  │ Torgersen │  2007 │
+│ Adelie  │ Torgersen │  2007 │
+│ Adelie  │ Torgersen │  2007 │
+│ Adelie  │ Torgersen │  2007 │
+│ Adelie  │ Torgersen │  2007 │
+│ Adelie  │ Torgersen │  2007 │
+│ Adelie  │ Torgersen │  2007 │
+│ Adelie  │ Torgersen │  2007 │
+│ …       │ …         │     … │
+└─────────┴───────────┴───────┘
+```
+
+Notice that the table is a little too wide to display all the columns now (depending on your screen-size). `bill_length` is now present in millimeters AND centimeters. Use a `select` to trim down the number of columns we're looking at.
+
+```python
+penguins.mutate(bill_length_cm=penguins.bill_length_mm / 10).select(
+    "species",
+    "island",
+    "bill_depth_mm",
+    "flipper_length_mm",
+    "body_mass_g",
+    "sex",
+    "year",
+    "bill_length_cm",
+)
+```
+
+```html
+┏━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━┳━━━━━━━━━━━━━━━━┓
+┃ species ┃ island    ┃ bill_depth_mm ┃ flipper_length_mm ┃ body_mass_g ┃ sex    ┃ year  ┃ bill_length_cm ┃
+┡━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━╇━━━━━━━━━━━━━━━━┩
+│ string  │ string    │ float64       │ int64             │ int64       │ string │ int64 │ float64        │
+├─────────┼───────────┼───────────────┼───────────────────┼─────────────┼────────┼───────┼────────────────┤
+│ Adelie  │ Torgersen │          18.7 │               181 │        3750 │ male   │  2007 │           3.91 │
+│ Adelie  │ Torgersen │          17.4 │               186 │        3800 │ female │  2007 │           3.95 │
+│ Adelie  │ Torgersen │          18.0 │               195 │        3250 │ female │  2007 │           4.03 │
+│ Adelie  │ Torgersen │           nan │              NULL │        NULL │ NULL   │  2007 │            nan │
+│ Adelie  │ Torgersen │          19.3 │               193 │        3450 │ female │  2007 │           3.67 │
+│ Adelie  │ Torgersen │          20.6 │               190 │        3650 │ male   │  2007 │           3.93 │
+│ Adelie  │ Torgersen │          17.8 │               181 │        3625 │ female │  2007 │           3.89 │
+│ Adelie  │ Torgersen │          19.6 │               195 │        4675 │ male   │  2007 │           3.92 │
+│ Adelie  │ Torgersen │          18.1 │               193 │        3475 │ NULL   │  2007 │           3.41 │
+│ Adelie  │ Torgersen │          20.2 │               190 │        4250 │ NULL   │  2007 │           4.20 │
+│ …       │ …         │             … │                 … │           … │ …      │     … │              … │
+└─────────┴───────────┴───────────────┴───────────────────┴─────────────┴────────┴───────┴────────────────┘
+```
+
+### selectors
+
+Typing out ALL of the column names _except_ one is a little annoying. Instead of doing that again, we can use a `selector` to quickly select or deselect groups of columns.
```

Comment:
ALL should be _emphasized_ with as `_all_` instead of capitalized which makes it look like a keyword.